### DISCUSSION TITLE
SSD Space Requirements for SDK Environment

### DISCUSSION THREAD

**Discussion by thewhiterabbit** [saleor/saleor#6724] [Deployment]
I've looked everywhere and any question regarding the SSD Space requirements to install Saleor are ignored, or side stepped.

As a software engineer with over 3 decades of experience I know the answer is "dependent on your catalog data" when it comes to production, however if someone just wants to SDK install and will only add a few testing categories, products, etc., what do they need to reserve in a VPS instance?

I tried to use 10gb thinking that should be adequate, but no, it's not. In fact the installation froze here `Unzipping Cypress        64% 9s [title changed]`, so I imagine that there needs to be 25+ GB of SSD space reserved for the install/compilation for Saleor in an SDK environment.

Can anyone from the core team please establish the minimum space requirements for an SDK install?

Very much appreciated, and this product has been a long overdue implementation in the eCommerce realm for decades.

Thank you for making Saleor.

**Comment by krzysztofwolski** [comment]
You don't need to install frontends on your production. You can check how our GH Actions CI configuration works - CI builds artifacts and pushes them to the s3 bucket which also hosts them. Or push to Vercel, in the case of the storefront.
